recipes/pybind11/all/test_package/CMakeLists.txt

```diff
@@ -1,8 +1,6 @@
 cmake_minimum_required(VERSION 2.8.11)
 project(test_package CXX)
 
-set(CMAKE_VERBOSE_MAKEFILE TRUE)
```

Comment:
You remove this from every recipe you editing

We all should probably agree if we use that or not for stringency. Pinging some people for opinions // @danimtb @SSE4 @uilianries @ericLemanissier

Reply:
we have discussed about it in the past, but I can't find the conversation (so many notifications, you know). I'll create new issue in the hooks repo.

Reply:
https://github.com/conan-io/hooks/issues/188